src/deprecated.jl

```diff
@@ -1,3 +1,5 @@
+export by, aggregate
```

Comment:
I moved it here as otherwise we might forget to clean up these when we remove the functions.